qiita_db/util.py

```diff
@@ -714,7 +714,17 @@ def path_builder(db_dir, filepath, mountpoint, subdirectory, obj_id):
                 for fpid, fp, fp_type_, m, s in results]
 
 
-def purge_filepaths():
+def _rm_files(TRN, fp):
```

Comment:
From looking at the file, I think this is because you need to import the TRN object, right @josenavas?